### DISCUSSION TITLE
Issues with Starscream version 4.0.4

### DISCUSSION THREAD

**Discussion by Lvykk** [WalletConnect/walletconnect-monorepo#1671] [Developer Support]
When the Starscream version is 4.0.4, after I use WebSocket to connect, after 300 seconds, the disconnection operation is triggered,(disconnected("Disconnecting for load balancing reasons", 4010))

**Comment by llbartekll** [comment]
hey @Lvykk we have the reconnection logic implemented and going to release the changes today, please upgrade your version to v1.0.6
